src/api/nodeRpcCall.js

```diff
@@ -1965,16 +1975,26 @@ const getAllContracts = async () => {
     const contractId = id.args[0];
     const judgesSignatures = getSignaturesForGivenId(rawJudgesSignatures, contractId);
     const partiesSignatures = getSignaturesForGivenId(rawPartiesSignatures, contractId);
-
     const contract = maybeContract.unwrapOr(null);
+
+    let data = null;
+    if (contract) {
+      try {
+        const hexData = contract.data.toString('hex');
+        data = Buffer.from(pako.inflate(hexToU8a(hexData))).toString('utf-8');
```

Comment:
Never copy-paste such code. Extract it and deduplicate with `getSingleContract`.